packages/react-native/src/private/webapis/performance/UserTiming.js

```diff
@@ -36,7 +36,7 @@ export class PerformanceMark extends PerformanceEntry {
     super({
       name: markName,
       entryType: 'mark',
-      startTime: markOptions?.startTime ?? performance.now(),
```

Comment:
I don't know if it's in the spec, but when creating a new `PerformanceMark` instance in Chrome, `startTime` is now by default, not 0.